<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>430</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Count!</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="4">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="skipPushButton">
      <property name="font">
       <font>
        <family>Sans</family>
       </font>
      </property>
      <property name="text">
       <string>Skip</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="submitPushButton">
      <property name="font">
       <font>
        <family>Sans</family>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="font">
       <font>
        <family>Sans</family>
       </font>
      </property>
      <property name="text">
       <string>Waiting for answer...</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="questionLabel">
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2 x 2 = ?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="answerTextEdit">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter the answer...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newGameAction"/>
    <addaction name="settingsAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuInformation">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInformation"/>
  </widget>
  <action name="newGameAction">
   <property name="text">
    <string>New game</string>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
    </font>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
    </font>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
